src/index.html

```diff
@@ -11,7 +11,65 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label for="surname">Surname: <input type="text" autocomplete="off"></label>
+          <label for="name">Name: <input type="text" autocomplete="off"></label>
```

Comment:
let's format your code a bit
```suggestion
          <label for="surname">
            Surname: 
            <input 
              type="text"
              autocomplete="off"
              add name field here
            >
          </label>
          <label for="name">Name: <input type="text" autocomplete="off"></label>
```

<img width="332" alt="image" src="https://user-images.githubusercontent.com/102411130/228448602-a55be33e-68e1-4645-8c85-950228f77a6e.png">

Reply:
requirements
<img width="279" alt="image" src="https://user-images.githubusercontent.com/102411130/228448372-3a60cea9-8259-443e-aa78-20b82f384196.png">